client/dashboard/domains/domain-overview/featured-card-emails.tsx

```diff
@@ -0,0 +1,50 @@
+import { Icon } from '@wordpress/components';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { envelope } from '@wordpress/icons';
+import { Domain } from '../../data/domain';
+import OverviewCard from '../../sites/overview-card';
+import type { Mailbox, EmailProvider } from '../../data/emails';
+
+const getAccountTypeLabel = ( accountType: EmailProvider ) => {
+	switch ( accountType ) {
+		case 'google_workspace':
+			return __( 'Google Workspace' );
+		case 'forwarding':
+			return __( 'Forwarding' );
```

Comment:
Related to the previous comment, I was not able to reach this case.

```suggestion
		case 'email_forwarding':
			return __( 'Forwarding' );
```